bigtable/admin.go

```diff
@@ -1457,14 +1457,21 @@ func UpdateInstanceAndSyncClusters(ctx context.Context, iac *InstanceAdminClient
 
 // RestoreTable creates a table from a backup. The table will be created in the same cluster as the backup.
 func (ac *AdminClient) RestoreTable(ctx context.Context, table, cluster, backup string) error {
-	ctx = mergeOutgoingMetadata(ctx, ac.md)
-	prefix := ac.instancePrefix()
-	backupPath := ac.backupPath(cluster, backup)
+	return ac.RestoreTableFrom(ctx, ac.instance, table, cluster, backup)
+}
 
+// RestoreTableFrom creates a new table to a different instance by restoring from this completed backup.
```

Comment:
I think the docs could still be a little clearer. What is "this" here?

My understanding is: the admin client is tied to a specific instance `foo`. In the latest version of this change, we take a table _from_ the source instance/cluster and restore it to a new table in `foo`. Is that correct?

Reply:
yes sorry this was leftover docs that I have now fixed. You are correct